lib/component.js

```diff
@@ -317,6 +317,8 @@ class Component extends WebComponent {
     } else {
       this.el = this;
     }
+
+    this.slowThreshold = 20;
```

Comment:
```suggestion
    this.slowThresholdMs = 20;
```
is the idea of making this an instance variable that someone could configure it per component by constructing a component and then modifying `myComponent.slowThreshold` before connecting it to the DOM? maybe it would be better as a `config` value with a default?